spec/fixtures/go/go-hello.go

```diff
@@ -27,4 +27,27 @@ func (conf Config) Access(kong *pdk.PDK) {
 		message = "hello"
 	}
 	kong.Response.SetHeader("x-hello-from-go", fmt.Sprintf("Go says %s to %s", message, host))
+	kong.Ctx.SetShared("shared_msg", message)
+}
+
+func (conf Config) Log(kong *pdk.PDK) {
+    access_start, err := kong.Nginx.GetCtxFloat("KONG_ACCESS_START")
```

Comment:
Indentation on this function seem incorrect (four spaces instead of tabs)

Reply:
:facepalm: